site/src/i18n/en/usersPage.json

```diff
@@ -5,5 +5,15 @@
   "deleteMenuItem": "Delete",
   "listWorkspacesMenuItem": "View workspaces",
   "activateMenuItem": "Activate",
-  "resetPasswordMenuItem": "Reset password"
+  "resetPasswordMenuItem": "Reset password",
+  "editUserRolesTooltip": "Edit user roles",
+  "fieldSetRolesTooltip": "Available roles",
+  "roleDescription": {
+    "owner": "Owner can manage all resources, including users and their permissions",
```

Comment:
I tried a screen reader and it read both the title and the description, so each role name was said twice in a row. Maybe start the descriptions with "Can."

Reply:
I saw that right after clicking on merge 🤦‍♂️ I think this is a good suggestion.